resources/lib/ui/login.py

```diff
@@ -0,0 +1,98 @@
+# -*- coding: utf-8 -*-

```

Comment:
please move into `NineAnimeBrowser` also note it has an internal member holding domain,
when we add the feature of changing domain via settings this will be broken..